core/jobs/src/main/java/com/cognifide/aet/job/common/comparators/requestmonitoring/utils/RequestMonitoringResultBuilder.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * AET
+ * <p>
+ * Copyright (C) 2013 Cognifide Limited
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.job.common.comparators.requestmonitoring.utils;
+
+public class RequestMonitoringResultBuilder {
```

Comment:
I consider this pattern is not needed in this context - correct me if I'm wrong but this builder is used only in one place and all of the fields seem to be mandatory. What is more - this is very simple object (2 fields) - I don't see a need to add this - constructor of `RequestMonitoringResult` should be sufficient.

Reply:
In principle I agree, I was a little bit reluctant on adding the logic of conversion from bytes to kB to the data object (RequestMonitoringResult). Actually, that was the only reason I decided to introduce this pattern.